Algorithms/JavaScript/digital-root.js

```diff
@@ -4,13 +4,13 @@
  */
 
 var addDigits = function(num){
-    if (num < 10) return num;
-    if (num % 9 === 0) return 9;
-    var sum = 0;
-    num = num.toString().split("");
-    for (var i = 0; i < num.length; i++){
-        sum += Number(num[i]);
-    }
-    return sum % 9;
+  if (num < 10) return num;
+  if (num % 9 === 0) return 9;
+  var sum = 0;
+  num = num.toString().split("");
```

Comment:
This can also be done mathematically using `% 10` and `/ 10` instead of turning it into a string.